### DISCUSSION TITLE
Explanation on how to setup this Integration  without a Mini for Energy Page

### DISCUSSION THREAD

**Discussion by thorrrr** [BottlecapDave/HomeAssistant-OctopusEnergy#271] [General]
Hi
Sorry if i missed it but can you explain how i use this integration because you seem to mention it only works properly if you have Octopus Mini (which are as rare a rocking horse poo) been waiting 8 months:(

But i am currently  on Flexible Nov 2022 v1  with a gen 1 smart meter that works.
But the sensors you mention in the docs giver a different price i get from Octopus App 
Plus i also get a different price from  Bright compared to this app!

So can this app be used with the Energy dashboard without a mini?

Also i am looking at moving onto the new July 2023 flexible daily tariff will this app cover this?

PS Do you know how to get a mini quicker

**Comment by BottlecapDave** [comment]
> Sorry if i missed it but can you explain how i use this integration because you seem to mention it only works properly if you have Octopus Mini (which are as rare a rocking horse poo) been waiting 8 months:(

This integration works with all tariffs, but [some features](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md#home-mini) are only available if you have a Home Mini or in the future are on a particular tariff.

> But the sensors you mention in the docs giver a different price i get from Octopus App
Plus i also get a different price from Bright compared to this app!

What are the differences in prices? Do the prices differ between OE app ad Bright? The values returned by OE don't include any government price caps, so you'll need to follow [these instructions](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md#government-pricing-caps) if your rate exceeds these values for your area. There are some issues with tracker tariffs where data is available in two different places where I try one location first and then the other.

> So can this app be used with the Energy dashboard without a mini?

As per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md) you can use this integration with the energy dashboard, but the frequency of the data you receive depends on if you have a Home Mini.

> Also i am looking at moving onto the new July 2023 flexible daily tariff will this app cover this?

This integration automatically picks up any new tariff provided by OE, so it _should_ work. It is currently used by people who are on tracker tariffs. It sometimes has issues with new tariffs if they're in beta. And it will have issues with tariffs where OE decide to provide the data in a different way.

> PS Do you know how to get a mini quicker

I'm afraid not. There's a long list of people in the OE forums who are waiting :(

**Comment by baileytv** [comment]
Not sure how/if I should jump on this, but I set my config up without a Mini. I now have one which arrived yesterday, I can't see how/where I can update the config to now reflect that one is available to it?

**Reply by BottlecapDave** [reply]
Hi there. You can go into the integration and tap configure next to the entry called "account" (it might be named "octopus energy" depending on how long ago you initially set up the integration). You should then be able to check the box and submit to gain the new sensors

**Comment by dodavies** [comment]
If you can't get a Mini perhaps change the IHD for a glow mqtt IHD
https://shop.glowmarkt.com/products/display-and-cad-combined-for-smart-meter-customers
